File: lib_common/src/main/java/com/wujia/lib_common/data/network/HttpCallBack.java
package com.wujia.lib_common.data.network;

/**
 * Created by xmren on 2018/3/9.
 */

public interface HttpCallBack<T extends BaseResponse> {
    void onStart();

    void onFailed(String code, String message);

    void onResponse(T response);
}

File: lib_common/src/main/java/com/wujia/lib_common/data/network/HttpCallBack.kt
package com.wujia.lib_common.data.network

/**
 * Created by xmren on 2018/3/9.
 */

interface HttpCallBack<T : BaseResponse> {
    fun onStart()

    fun onFailed(code: String, message: String)

    fun onResponse(response: T)
}
